frontend/app/assets/stylesheets/spree/frontend/views/spree/layouts/spree_application.scss

```diff
@@ -1,3 +1,6 @@
+body {
```

Comment:
We don't need this, we need to change this here:

1. https://github.com/spree/spree/blob/babfd2deed7c5adc474e3068380d0138f6febe20/frontend/app/assets/stylesheets/spree/frontend/views/spree/shared/category_nav_bar.scss#L79
2. https://github.com/spree/spree/blob/b0814469b0df3603b8567edb20de1e686c3e5855/frontend/app/assets/stylesheets/spree/frontend/variables/bootstrap-overrides.scss#L23